app/jinja_filters.py

```diff
@@ -295,7 +319,11 @@ def __init__(self, option, index, answer):
 
 
 class OtherConfig:
-    def __init__(self, detail_answer_field, detail_answer_schema):
+    def __init__(
+        self,
+        detail_answer_field: SelectFieldBase._Option,
+        detail_answer_schema: Mapping,
```

Comment:
Mapping needs top level parameters (as described in type hinting doc in runner).